### PR TITLE
Ensure storybook stories are not changed by PRs

### PR DESCRIPTION
**Pull request by cellog**
# Description

We have had several PRs subtly change storybook stories. This PR adds a new test based on [storyshots](https://github.com/storybooks/storybook/tree/next/addons/storyshots/storyshots-core) that creates a snapshot of all the stories. If any story changes output, it will fail the snapshot test.

In the process of developing this, many subtle errors in stories and one critical but subtle error in the css of the demo page were uncovered.

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [X] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [X] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [X] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by julien51** [comment]
Did you rebase? I see changes which I think were in another PR?

File: unlock-app/.jest/.eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  rules: {
+    'import/no-extraneous-dependencies': [
```
Comment (julien51) [inline unlock-app/.jest/.eslintrc.js]: Wait, wasn't that part of another PR? Did you rebase?

Comment (julien51) [reply unlock-app/.jest/.eslintrc.js]: HA! 1 PR 1 purpose ;) do you mind moving that to another one? I know this is annoying but it makes things so much easier down the line.

Comment (cellog) [reply unlock-app/.jest/.eslintrc.js]: yep, marked as WIP so no one is tempted to approve it

File: unlock-app/package.json [hunk 2]
```diff
@@ -120,4 +120,4 @@
     "prettier-eslint-cli": "^4.7.1",
     "react-testing-library": "^5.2.3"
   }
-}
+}
```
Comment (julien51) [inline unlock-app/package.json]: 🤪

File: unlock-app/src/pages/demo.js [hunk 3]
```diff
@@ -99,8 +99,8 @@ const GlobalStyle = createGlobalStyle`
     @import url('https://fonts.googleapis.com/css?family=Source+Serif+Pro:400,700|UnifrakturCook:700');
 
     body {
-      background - color: #fdfaf7;
-      font - family: 'Source Serif Pro', serif;
```
Comment (julien51) [inline unlock-app/src/pages/demo.js]: wow. this was bad.

**Pull request merged**